src/commands/fun/latex.ts

```diff
@@ -0,0 +1,86 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment, BaseGuild } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		},
+		{
+			name: 'transparent',
```

Comment:
Would this be needed? Making it false would render the latex expression unreadable to light mode users (as we discussed earlier). The background you've used seems to blend perfectly with dark mode, anyways

Reply:
if you want to copy-paste it to a different site, it'll have an ugly grey background on it

Reply:
could potentially add in a URL to the embed that links to the transparent image? Change the embed declaration to something like this, if you wish:

```
const embed = new MessageEmbed()
    .setImage('attachment://tex.png')
    .addFields(
	{ name: 'Link to transparent image', value: '<URL here>' },
    );
```

Reply:
removed transparent background feature to make sure everyone can always see the image clearly